pkg/reconciler/reconcile.go

```diff
@@ -9,7 +9,35 @@ import (
 	"go.uber.org/zap"
 )
 
-func Reconcile(logger *zap.Logger, client *db.Database, req forman.Request) forman.Result {
+type FleetReconciler struct {
+	// Data required for reconcile phases
+}
+
+func (r *FleetReconciler) initReconciler(ctx context.Context) {
+	// Initializes the fields of FleetReconciler struct
+}
+
+func (r *FleetReconciler) reconcilePhases() (forman.Result, error) {
+	// Reconciles the desired state
+	// Example for an installation flow:
+	// 	1. Ensures the infra via AWS Cloudformation
+	// 	2. Ensures ROKS Cluster installation on existing infra
+	// 	3. Ensures ManagedFusionAgent Operator on ROKS Cluster
+	// 	4. Ensures presence of config for MFAO monitoring stack
+	// 	5. Ensures Data Foundation Config is present on ROKS Cluster
+	return forman.Result{}, nil
+}
+
+func (r *FleetReconciler) Reconcile(ctx context.Context) (forman.Result, error) {
+	r.initReconciler(ctx)
+	result, err := r.reconcilePhases()
+	if err != nil {
+		return forman.Result{}, err
+	}
+	return result, nil
+}
```

Comment:
This entire function is equivalent to:
```
func (r *FleetReconciler) Reconcile(ctx context.Context) (forman.Result, error) {
	r.initReconciler(ctx)
	return r.reconcilePhases()
}
```

As I already mentioned that there is not yet a reason to have `initReconciler` in a prev comment then I would say it is equivalent to 
```
func (r *FleetReconciler) Reconcile(ctx context.Context) (forman.Result, error) {
	return r.reconcilePhases()
}
```

Which means we don't need r.reconcilePhases() and we can just fold this into Reconcile.

Reply:
done.